Publisher/nl/email-editor-drag-and-drop-templates.md

```diff
@@ -25,16 +42,12 @@ Er zijn verschillende blok-opties voor je nieuwsbrief:
 | Video                 | Optie om een video toe te voegen                                                                        |
 | Sociaal               | Optie om social media toe te voegen                                                                     |
 | Banner                | Optie om een banner toe te voegen                                                                       |
-| Timer                 | Optie om een aftel-timer toe te voegen                                                                  |
 | Menu                  | Optie om een menu toe te voegen                                                                         |
 | Html                  | Optie om zelfgeschreven HTML (en CSS) toe te voegen                                                     |
-| Carrousel \*          | Optie om meerdere afbeeldingen op één plek te tonen                                                     |
-| Accordeon \*          | Optie om content open en dicht te tonen waarbij de ontvanger het kan in- of uittklappen                 |
-| Formulier \*          | Optie om een formulier te tonen in je nieuwsbrief. Hierdoor hoeft je ontvanger niet naar een webpagina  |
 
 Elk blok heeft eigen opties om de weergave van het specifieke blok aan te passen.
 
-**\*** Dit werkt enkel in clients met AMP-ondersteuning. Bij clients zonder AMP-ondersteuning zal dit blok niet worden getoond. 
-
 ### Modules
+Elk **lijn-** of **structuurelement** kun je binnen je template opslaan als _module_. Hiermee kun je een vast element opslaan zodat je deze niet in ieder template opnieuw hoeft op te zetten. Je kunt een module maken door met je muis over het element te gaan en te klikken op de drie puntjes. Hier vind je de optie '_opslaan als module_'.
```

Comment:
Dit is de eerste keer dan wij de termen lijn en structuurelement horen. Beter hierboven aangeven wat het is. 
Bij de uitleg over regels, blokken en containers bijvoorbeeld.